### PR TITLE
Ros2 v0.8.0 fix packages2

### PR DESCRIPTION
**Pull request by tkimura4**
Fixed the cause of /autoware/state becoming Emergency

# How to check
```
$ros2 launch autoware_launch planning_simulator.launch.xml vehicle_model:=lexus vehicle_id:=default sensor_model:=aip_xx1 map_path:=[MAP_PATH]
Set initial pose and goal pose
$ros2 topic echo /autoware/state # check state
```

### CODE REVIEW

File: system/autoware_error_monitor/config/diagnostic_aggregator/sensing.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
         lidar:
           type: 'diagnostic_aggregator/AnalyzerGroup'
-          path: velodyne
+          path: lidar
```
Comment (kenji-miyake) [inline system/autoware_error_monitor/config/diagnostic_aggregator/sensing.yaml:21]: You should add an AnalyzerGroup.
We need `/sensing/lidar/velodyne/*` and `/sensing/lidar/livox/*`

Comment (tkimura4) [reply system/autoware_error_monitor/config/diagnostic_aggregator/sensing.yaml:21]: Thanks. Fixed at  261fc14

**Review by kenji-miyake — APPROVED** [review]
LGTM

**Comment by k0suke-murakami** [comment]
Could you resolve conflicts?

**Comment by tkimura4** [comment]
rebased on ros2-v0.8.0-beta

**Review by k0suke-murakami — APPROVED** [review]
LGTM

**Pull request merged**